Dockerfile.dapper

```diff
@@ -32,8 +32,13 @@ RUN if [ "${ARCH}" == "amd64" ]; then \
         curl -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/spectrometer/master/install.sh | sh; \
     fi
 
-# install controller-tools for crd generation
-RUN go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.12.0
+# Tool for CRD generation.
+ENV CONTROLLER_GEN_VERSION v0.12.0
```

Comment:
If we are making this change, should we move all the `ENV` variables to the top to keep it consistent?

Reply:
Sure, that might be good. I only extracted this as an ENV not because it's to be configurable, but to allow dependency bump tools notify us that there is a newer version.
Looks like the Renovate bot does inspect .dapper files - https://github.com/rancher/rancher/pull/41453. This one is just inline curl fetch of a binary.
Not sure why it hasn't notified us of a Kustomize or this controller-gen bumps, for example. Perhaps because they are specified in the ENV directives, actually, rather than inline. Will investigate.

Reply:
For info Renovate inspect .dapper because there is a rule defined [here](https://github.com/rancher/renovate-config/blob/0596cc041373a388718f6364b9ae12e2a82a635a/default.json#L89)

Reply:
Thanks! Will add a rule once we merge this PR.